File: wwwroot/js/ProjectClass/FormingInvoice.js
import { PairList } from "./PairDetails.js";
export class FormingInvoice {
    InvoiceNumber;
    InvoiceDate;
    Buildings;
    ToBeInvoiced;
    InvoiceCharge;

    constructor() {
        this.InvoiceNumber = "";
        this.InvoiceDate = "";
        this.Buildings = new PairList();
        this.ToBeInvoiced = new PairList();
        this.InvoiceCharge = 0;
    }
}

File: Data/Project Classes/FormingInvoice.cs
namespace SJC_Portal.Data.Project_Classes
{
    public class FormingInvoice
    {
        public int? InvoiceNumber { get; set; }
        public string? InvoiceDate { get; set; }
        public PairList? Buildings { get; set; }
        public PairList? ToBeInvoiced { get; set; }
        public double? InvoiceCharge { get; set; }
    }
}
